pkg/tracing/tracing.go

```diff
@@ -35,6 +36,14 @@ type contextKey struct{}
 // LogField is the key in log message field that holds tracing id value.
 const LogField = "traceid"
 
+const (
+	// TraceContextHeaderName is the http header name used to propagate tracing context.
+	TraceContextHeaderName = "trace-id"
```

Comment:
@janos @agazso @vojtechsimetka @nugaon do you have any preference for the header name? should we go with `Trace-Id`? `Swarm-Trace-Id`? any other ideas?

Reply:
I like `swarm-trace-id` for the consistency with other custom headers.

Reply:
👍

Reply:
`swarm-trace-id` is good as it is analogue to default value `uber-trace-id` https://github.com/jaegertracing/jaeger-client-go/blob/master/constants.go#L58 and it has to be in lower case as per comment https://github.com/jaegertracing/jaeger-client-go/blob/master/header.go#L33.

Reply:
@janos I didn't know it has to be lowercase, but in the HTTP standard the header name is case-insensitive so I just prefer to refer to header names in lowercase :)

https://tools.ietf.org/html/rfc7230#section-3.2

Reply:
@agazso I did not know either, I've just fount this int the comments in the library that is used and found this interesting fact. I thought that it would be good to share it here mainly because of the case-insensitivity you mentioned. It looks like just a specific requirement for this particular library.

Reply:
very strange :)

Reply:
Yes, the library requires lowercase values.